### DISCUSSION TITLE
Coral custom model creation

### DISCUSSION THREAD

**Discussion by H1ghSyst3m** [blakeblackshear/frigate#14752] [Ask A Question]
Hi Guys,
I have a question. Right now I'm using a fine tuned Frigate+ Model.
It works really well and detects cat every time, but the problem is, that it also detects other cats. 
I don't want to make false detections in the frigate+ model so that it only detects my cat as a cat, because it's also later used in the base model and would increase false reports.

Can someone tell me how to use and create a custom model for Frigate coral TPU?

**Comment by hawkeye217** [comment]
This won't work the way you're envisioning - the models used by Frigate are object detection models, not classification models. These are two different animals (pun intended).

**Reply by H1ghSyst3m** [reply]
Won't Tensorflow or PyTorch models not work. I really just need the detection, to detect my cat. Nothing else

**Reply by hawkeye217** [reply]
https://labelyourdata.com/articles/object-detection-vs-image-classification
